dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -496,7 +496,8 @@ public void update(Context context, Item item) throws SQLException, AuthorizeExc
 
         super.update(context, item);
 
-        // Set sequence IDs for bitstreams in item
+        // Set sequence IDs for bitstreams in Item. To guarantee uniqueness,
+        // sequence IDs are assigned in sequential order (starting with 1)
```

Comment:
this comment requires me to check the code more closely as I read starting with 1 and the next line is just sequence = 0...
it is actually right but not obvious.
I suggest to put here the following comment

// discover the max sequence ID in use

and move/merge the current one to line 515

Reply:
Hi @abollini : Sorry, I don't understand your request here.  the "discover the max sequence ID in use" comment already exists (in a slightly different wording) on line 504: https://github.com/DSpace/DSpace/blob/16117bf132a6e2410dba2c501cffed6e5cc1af0e/dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java#L504

So, the purpose of the comments I added here were to summarize what is going in in lines 501-528: https://github.com/DSpace/DSpace/blob/16117bf132a6e2410dba2c501cffed6e5cc1af0e/dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java#L501-L528

It's meant as more of a summary of a section of code.  That said, I agree the code structure itself is a bit confusing, and maybe this entire section of code should be a separate private method.  In any case, I'd recommend we leave code changes to a separate PR. 

That said, I'll make a minor update to this specific comment to make it clear that it's commenting on a section of code rather than the next line (which says `sequence = 0`).